lib/index.js

```diff
@@ -16,30 +16,45 @@ var Yandex = module.exports = integration('Yandex Metrica')
   .assumesPageview()
   .global('yandex_metrika_callbacks')
   .global('Ya')
+  .option('type', 0)
   .option('counterId', null)
   .option('clickmap', false)
   .option('webvisor', false)
+  .option('trackHash', false)
+  .option('trackLinks', false)
+  .option('accurateTrackBounce', false)
```

Comment:
Actually should be fine since that's the yandex default as well https://cloudup.com/cW0cC7VA51q.